cmake/linux/helpers.cmake

```diff
@@ -60,7 +61,7 @@ function(set_target_properties_obs target)
       ${target}
       PROPERTIES
         VERSION 30
```

Comment:
I honestly no idea if this one is supposed to be the frozen value or the real value if you happen to know @tytan652

Reply:
`VERSION` and `SOVERSION` needs to be identical to have only a share object without symlink (`.so.31.0.0` to .so.30), mainly to avoid linking issue.

`VERSION` and `SOVERSION` will re-become *different* once `set_target_properties_obs` and how we manage versioning is reworked.

Reply:
Ok I set them both as the variable. The primary reason i wanted to use this is to mark all the places we need to cleanup with the `set_target_properties_obs` changes, not that we need to keep this variable around for any real usage.

Reply:
I wouldn't sweat over that too much by this point - the actual necessary changes have neither been made nor have I seen any actual work on them, meaning that the internal versioning of `libobs` is still manually and independently maintained in a header file and at any point can (and probably will) deviate from this variable.

It's all just cosmetics until a proper change lands that introduces a proper "libos Library Version" variable to the build system which is then used for the `SOVERSION` as well as the generated `libobs` version header.

(I also expect some plugins and other internal code to treat the `libobs` version as the "program version", so those places would need to be fixed as well).